runconfig/hostconfig.go

```diff
@@ -272,7 +273,7 @@ type HostConfig struct {
 	CPUQuota         int64            `json:"CpuQuota"` // CPU CFS (Completely Fair Scheduler) quota
 	BlkioWeight      int64            // Block IO weight (relative weight vs. other containers)
 	OomKillDisable   bool             // Whether to disable OOM Killer or not
-	MemorySwappiness int64            // Tuning container memory swappiness behaviour
+	MemorySwappiness *big.Int         // Tuning container memory swappiness behaviour
```

Comment:
Why `*big.Int` rather than `*int64`? There's an additional (significant) cost to `big.Init` as far as I know.